bucket/tern-subtitle-file-translator.json

```diff
@@ -0,0 +1,24 @@
+{
+    "homepage": "https://tern.1c7.me",
+    "description": "Translate .srt .ass .vtt file",
+    "license": {
+        "identifier": "Shareware",
+        "url": "https://tern.1c7.me/pricing"
+    },
+    "version": "1.1.1",
+    "url": "https://github.com/1c7/Translate-Subtitle-File/releases/download/1.1.1/win.zip",
```

Comment:
Filename change in latest release is intentional?

`Windows-$version.zip` -> `win.zip`

Reply:
I have communicated with the author, he will use a fixed name in the future, the file name of the latest version has been changed to `Tern_Windows_1.1.1.zip`.